src/bin/http_server.rs

```diff
@@ -37,6 +37,31 @@ async fn main() -> IoResult<()> {
         .await
         .expect("Failed to create AWS S3 configuration");
 
+    match cli.mode {
+        RunMode::Server => run_server(db, cerved_qrp_client, s3_client).await,
+        RunMode::Daemon => run_daemon(db, cerved_qrp_client, s3_client).await,
+    }
+}
+
+async fn run_daemon(db: Db, cerved_qrp_client: CervedQrpClient, s3client: S3Client) -> IoResult<()> {
+    info!("Ceres C'è: starting as Daemon");
+    let deferred_qrp_service = DeferredQrpService {
+        db: db.clone(),
+        qrp_client: cerved_qrp_client.clone(),
+        s3_client: s3client.clone(),
+    };
+
+    let mut interval = tokio::time::interval(std::time::Duration::from_secs(FIFTEEN_MINUTES_IN_SECONDS));
+    loop {
+        interval.tick().await;
+        let _ = deferred_qrp_service.process_deferred_qrp().await;
```

Comment:
See, I always find one last thing.

`process_deferred_qrp` returns a result. By discarding it, you're doing the rust equivalent of swallowing an exception in java.
```suggestion
        deferred_qrp_service.process_deferred_qrp().await?;
```

Reply:
Ok now I see why I took the IDE suggestion to add `let _ =`: the function returns an `IoResult`, I have to convert the `anyhow` into it in some way. Any idea?

Reply:
I've resolved using `anyhow::Result` as return typw for the main fn, and I mapped the error from actix `HttpServer::new`